adagrams/game.py

```diff
@@ -1,11 +1,142 @@
 def draw_letters():
-    pass
+    """
+    import string package to import letters.
+    import random to select random letters from a list
+    import repeat to add repetative letters to our list
+    import sample to select each letter only once
+    """
+    import string
+    import random
+    from itertools import repeat
+    from random import sample
+    """
+    create a list of letters
+    """
+    letters = list(string.ascii_uppercase)
+    """
+    create a frequency list in order
+    """
+    frequency = [9,2,2,4,12,2,3,2,9,1,1,4,2,6,8,2,1,6,4,6,4,2,2,1,2,1]
+    """
+    iterate over the letters list and add repeated items using the frequency-1
+    """
+    for i in range(len(letters)):
+        letters.extend(repeat(letters[i],frequency[i]-1))
+    
+    """
+    obtain a sample of ten letters
+    """
+    selection =  sample(letters,10)
+    return selection
 
-def uses_available_letters(word, letter_bank):
-    pass
 
+    # pass
```

Comment:
In the future, you can delete `pass` altogether as it's just a placeholder once actual code is developed.